schema/CompoundSchema.mcf

```diff
@@ -0,0 +1,222 @@
+Node: dcid:ChemicalCompound
+name: "ChemicalCompound"
+typeOf: schema:Class
+subClassOf: schema:Thing
+description: "Chemical compound."
+sameAs: "https://www.ebi.ac.uk/chembl/"
```

Comment:
From schema.org/sameAs: "URL of a reference Web page that unambiguously indicates the item's identity. E.g. the URL of the item's Wikipedia page, Wikidata entry, or official website."

Given that, I'm puzzled by multiple nodes having the same sameAs in this file. Either we should provide uniquely identifying URLS, or drop sameAs, or change it to url (if that describes this).

Reply:
Sam completed this schema. We will discuss this with her in the meeting.

Reply:
Per morning discussion are we changing this to "url" ?